poker/settings.py

```diff
@@ -63,13 +63,20 @@
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
+# DATABASES = {
```

Comment:
We need to use the postgres in development environment as well.

Reply:
Fine , actually on my system I was facing some issues with postgres. I will correct this

Reply:
now I need to close this pr? as already this issue is merged by another PR